docs/decisions/0005-project-labels.md

```diff
@@ -0,0 +1,149 @@
+
+The following list aims to comprehensively document all our labels.
+
+### `area/*`
+
+The `area` category species is the user's visible area that is affected by the
+issue. It helps to find a duplicate issue during reporting or the triage
+process.
+
+An example is `area/kubernetes`.
+
+### `component/*`
+
+The `component/*` category specifies the part of the implementation that will
+need to be modified during the implementation.
+
+An example is `component/ui`.
+
+### `kind/*`
+
+This is the "document kind". Each issue **must** have exactly one `kind/*`
+label. The labels `kind/bug` and `kind/enhancement` are being standardized
+across SUSE Rancher projects.
+
+#### `kind/bug`
+
+This issue describes a defect in the software. Sometimes it requires a judgement
+call, but in general, incomplete functionality, or different behaviour from a
+similar project should be labelled as `kind/enhancement` instead.
+
+#### `kind/enhancement`
+
+This issue requests additional (or a change in) functionality. The request
+typically comes from a user or the PO.
+
+#### `kind/epic`
+
+An epic is an umbrella issue to implement a feature during a single release
+cycle. It includes a work-breakdown as a task list to individual `kind/story`
+issues.
+
+The epic should include acceptance criteria and link to any documentation and QA
+work required to finish the epic.
+
+Any unfinished tasks are not automatically rolled over into the next milestone
+at release time; they go back into the backlog and will have to be scheduled
+again by the triage team.
+
+#### `kind/story`
+
+A story is an individual work item to implement (part of) a feature. We do not
+use `kind/enhancement` for this to avoid diluting the reporting of open feature
+requests.
+
+#### `kind/quality`
+
+This issue is about refactoring, adding tests, improving CI. Any activity that
+improves the quality of the project, but is neither a bug fix nor a user visible
+feature.
+
+### `priority/*`
+
+Not every bug, feature or issue requires having a `priority/` label. We won't
+prioritize every ticket since triage is a long process for those actively
+participating in it, but it's important to highlight what needs to sorted out
+urgently.
+
+- priority/0 – Blocker: For bugs, this is an issue that requires immediate
+  attention and is a release blocker.
+- priority/1 – Critical/Urgent: For bugs, this is an issue that breaks existing
+  functionality but doesn't affect the primary use of the app and should be
+  addressed in the next release.
+- priority/2 – High priority: For bugs, this is an issue that should be triaged
+  amongst other higher priority work.
+
+### `regression`
+
+This label should only be applied to `kind/bug` issues and signifies that some
+functionality that was working in a previous release is now broken.
+
+### `release-note`
+
+This label is used to mark all issues within a milestone that should be
+explicitly mentioned in the release notes.
+
+### `triage/*`
```

Comment:
shouldn't there be a label representing something that has been triaged, so we don't keep going through issues that have already been triaged?

Reply:
I kind of assumed that once triaged, the issue gets its corresponding labels (ie `kind/*`, `area/*` etc). 

But I added a `triage/ready` label and corresponding text. Better be safe than sorry 😄